lib/topological_inventory/mock_source/operations/worker.rb

```diff
@@ -0,0 +1,75 @@
+require "topological_inventory/mock_source/logging"
+require "topological_inventory-api-client"
+
+module TopologicalInventory
+  module MockSource
+    module Operations
+      class Worker
+        include TopologicalInventory::MockSource::Logging
+
+        def initialize
+          self.payload = {}
+        end
+
+        def run
+          create_mock_source
+        end
+
+        private
+
+        attr_accessor :payload
+
+        def create_mock_source
+          source = post_source
+          post_endpoint(source)
+        end
+
+        def post_source
+          source = TopologicalInventoryApiClient::Source.new(
+            :name           => "Mock Source",
+            :source_type_id => '1',
+            :tenant_id      => '1',
```

Comment:
This has to be set with the x-rh-identity headers, again not depending on the id